project.clj

```diff
@@ -32,7 +32,7 @@
 
   :min-lein-version "2.7.1"
 
-  :parent-project {:coords [puppetlabs/clj-parent "0.4.1"]
+  :parent-project {:coords [puppetlabs/clj-parent "0.5.0-SNAPSHOT"]
```

Comment:
Seems like this may need to be rebased since the diff isn't showing the `io.dropwizard.metrics/metrics-core "3.1.2"` dependency that I see on the [current HEAD of the master branch](https://github.com/puppetlabs/puppetserver/blob/ff50c9cd40647ace5a94cc25499b1e20c7cddc62/project.clj#L51).  In any event, I think we should be able to remove the "3.1.2" and inherit the version from the forthcoming clj-parent release.